transactions/argument.go

```diff
@@ -22,7 +22,7 @@ func ArgAsCadence(a Argument) (cadence.Value, error) {
 	}
 
 	// Use cadence's own encoding library
-	c, err = c_json.Decode(j)
+	c, err = c_json.Decode(nil, j)
```

Comment:
[MemoryGauge](https://github.com/onflow/cadence/blob/432760449b2d2f234cc403d1c6252f30c925d9ab/runtime/common/metering.go#L34) has a single method MeterMemory that apparently can be used to pass information on memory usage back to the caller. As [UseMemory](https://github.com/onflow/cadence/blob/432760449b2d2f234cc403d1c6252f30c925d9ab/runtime/common/metering.go#L252) manages just fine with nil and memory usage information is not used, it can be left nil.